app/src/tunnel/kotlin/tunnel/FilterManager.kt

```diff
@@ -160,31 +178,47 @@ internal class FilterManager(
 
     private fun syncRules(ktx: Kontext) = {
         val active = store.cache.filter { it.active }
+        //TODO maybe clear clear() or remove() or onDestroy() !it.active here?
         val downloaded = mutableSetOf<Filter>()
+        val wildcard = store.cache.filter{it.wildcard}
+        wildcard.forEach {}
         active.forEach { filter ->
             if (!doValidateRulesetCache(filter)) {
                 ktx.v("fetching ruleset", filter.id)
                 ktx.emit(Events.FILTERS_CHANGING)
-                doFetchRuleset(doResolveFilterSource(filter), doGetMemoryLimit()).mapBoth(
-                        success = {
-                            blockade.set(ktx, filter.id, it)
-                            downloaded.add(filter.copy(lastFetch = System.currentTimeMillis()))
-                            ktx.v("saved", filter.id, it.size)
-                        },
-                        failure = {
-                            ktx.e("failed fetching ruleset", filter.id, it)
-                        }
-                )
+                if (filter !in wildcard) {
```

Comment:
Not sure what you mean. I thought this way made it cleaner to read.

Reply:
It's not that important to use different strings in the log because the filter-id tells you which list was saved or failed.
Thats why you can use `doFetchRuleset(doResolveFilterSource(filter), doGetMemoryLimit(), filter !in wildcard)`.